package.json

```diff
@@ -32,7 +31,8 @@
 		"numeral": "^2.0.6",
 		"parse-duration": "jellz/parse-duration",
 		"ts-node": "^8.8.2",
-		"typescript": "^3.8.3"
+		"typescript": "^3.8.3",
+		"xlsx": "^0.16.7"
```

Comment:
Does this need to be here anymore?

```suggestion
```

Reply:
Not sure why I didn't remove that, must've thought I did last night.